docs/proposals/batch-node-access.md

```diff
@@ -0,0 +1,217 @@
+## Motivation
+
+This proposal aims to solve the node access efficiency problem on the current KubeEdge edge computing platform.
+
+- The current KubeEdge only supports manual single-node access, which is cumbersome and time-consuming for large-scale deployment of edge nodes, and batch node access needs to be realized to improve efficiency.
+
+
+
+## Goals
+
+Batch access to edge nodes in the KubeEdge system:
+
+- Simplifies node access operations, improves deployment efficiency, and ensures system stability and reliability.
+
+
+
+## Proposal
+
+- **Configuration file support:** Supports configuration files in YAML format, containing information such as the CloudCore address, authentication token, KubeEdge version, etc. required for each node connection.
```

Comment:
This is just a comment. this does not block any development for this PR. (IMO this feature is great enhancement.)

quick question, does it support the configuration that could be different from each edge node? in other words, can user describe the physical node specific configuration different from each other such as `containerRuntime`? i believe that this is out of scope for this feature, right? this feature is more like focusing on `common` configuration and join the bunch of cluster in the same network all together?